benchmarks/tests/primary/src/tests/noop.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+import { createMemoryTest, neverOptimizeFunction } from "../helpers.js";
+
+const r = await createMemoryTest(neverOptimizeFunction(async () => {
+  return Math.random();
```

Comment:
What's the use of the noop?

Reply:
It lets me see how much resources are consumed as a baseline. It was especially helpful for tuning things like warmups and the gc logic